.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+**Is this pull request related to an existing issue? What is the issue number?**
+
+E.g. #47 (Creation of docs folder / corresponding docs site)
+
+**Description of proposed changes:**
+
+E.g. "This pull request implements a docs folder that provides documentation for the natlas project."
+
+**Should anyone specifically look at this?**
+
+E.g. "@0xdade should probably review this because he cares a lot about the documentation of the project."
```

Comment:
Perhaps also mention that you can request a review of someone with a PR and should direct it to those individuals for those that should specifically look at the project. This might be an edge case that we don't care about, but it may be worth adding grammar for.

Reply:
I'm not sure I follow what you're asking here. I am going to clarify wording to look for code owners if they exist (which we don't do right now, but may want to start doing in the future, especially as more specialized capabilities are created), though. What do you mean "request a review of someone with a PR and should direct it to those individuals for those that should specifically look at the project"?

Reply:
I was looking for a more streamlined way of stating that you can ask for a code review or to have the maintainer of a specific chunk of code review the PR. The language is cleaned up now in 99e82c24291b0bd36ec30c7ac587cf9cba2a0242.